### PR TITLE
{cae}[foss/2022a] PICI-LIGGGHTS v3.8.1 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/61956ca04c1694f75778d963293a0673 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/ca908c5abc28c0366deec8cc4ac91e93 for a full test report.

File: easybuild/easyconfigs/p/PICI-LIGGGHTS/PICI-LIGGGHTS-3.8.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'CMakeMake'
+
+name = 'PICI-LIGGGHTS'
+version = '3.8.1'
+
+homepage = 'https://github.com/uob-positron-imaging-centre/PICI-LIGGGHTS'
+description = """UoB Postron Imaging Centre's Improved LIGGGHTS distribution with
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PICI-LIGGGHTS/PICI-LIGGGHTS-3.8.1-foss-2022a.eb:8]:
```suggestion
description = """UoB Positron Imaging Centre's Improved LIGGGHTS distribution with
```

**Review by verdurin — CHANGES_REQUESTED** [review]

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
Looks fine.

**Pull request merged**